metadata.rb

```diff
@@ -4,7 +4,7 @@
 license           'Apache-2.0'
 description       'Installs Java runtime.'
 long_description  IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version           '1.50.0'
+version           '1.50.1'
```

Comment:
In general the versioning is left to the maintainers post acceptance for several reasons:
- we may disagree on the impact of the change and therefore how its versioned, in this case I would say its a minor not a patch
- PRs are not a FIFO queue, as such versioning them prior to acceptance leads to needing to ask people to rebase more often. For example the current version is 2.2.1